src/Runner.Listener/Runner.cs

```diff
@@ -370,7 +370,8 @@ private async Task<int> RunAsync(RunnerSettings settings, bool runOnce = false)
                 _term.WriteLine($"{DateTime.UtcNow:u}: Listening for Jobs");
 
                 IJobDispatcher jobDispatcher = null;
-                CancellationTokenSource messageQueueLoopTokenSource = CancellationTokenSource.CreateLinkedTokenSource(HostContext.RunnerShutdownToken);
+                CancellationTokenSource messageQueueLoopTokenSource =
```

Comment:
any chance you can undo these newline changes? make the diff hard to follow. 😢

Reply:
done!